x/bank/proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -77,15 +77,9 @@ service Query {
   // DenomMetadata queries the client metadata of a given coin denomination.
   rpc DenomMetadata(QueryDenomMetadataRequest) returns (QueryDenomMetadataResponse) {
     option (cosmos.query.v1.module_query_safe) = true;
-    option (google.api.http).get               = "/cosmos/bank/v1beta1/denoms_metadata/{denom}";
+    option (google.api.http).get               = "/cosmos/bank/v1beta1/denoms_metadata/{denom=**}";
```

Comment:
TIL, that's pretty cool, could you do that for all applicable queries?

Reply:
I looked into it, and it seems that except for the endpoint, there is no case where a slash comes in the path parameter.

Reply:
Cool, thanks!